src/UIComponents/ColumnInspector.jsx

```diff
@@ -106,7 +106,7 @@ class ColumnInspector extends Component {
           <form>
             <div>
               <label>
-                <div>{currentColumnData.id}</div>
```

Comment:
We need to keep this **and** add "Data Type:" below it.

Reply:
Added it back in!